### DISCUSSION TITLE
Unable to install Python Module in Win11 or Ubuntu 22LTS

### DISCUSSION THREAD

**Discussion by ssklare** [gaphor/gaphor#2103] [Q&A]
Hello,
I would really appreciate some help getting the Python module installed on Windows 11 and/or Ubuntu 22LST.
My goal is to use Python Jupyter Notebook as my main interaction method with Gaphor, but I keep getting stuck on some build dependencies for PyGObject.
In response I tried installing all seemingly relevant VC++ Build Tools, but still haven't had any luck.
Similar issues on an Ubuntu 22LTS VM.

Has anyone gotten past this issue?

![image](https://user-images.githubusercontent.com/13054928/221038415-28adc306-8d6c-4a83-b2d6-d30fc5267288.png)

**Comment by danyeaw** [comment]
Hi @ssklare, please take a look at our development environment setup instructions for [Windows](https://gaphor.readthedocs.io/en/latest/windows.html) and [Linux](https://gaphor.readthedocs.io/en/latest/linux.html).

**Reply by ssklare** [reply]
Thank you for responding @danyeaw!

I appreciate it and the work all the maintainers have done.

I'm sorry if I don't understand, but after reading through the setup instructions for the Windows and Linux dev environments - it doesn't look like this will install Gaphor as a Python module either. I am mistaken?

My end goal is being able to load and manipulate SysMl models in a Python Jupyter-Notebook, so I am specifically trying to get that module installed, not install Gaphor in general (which was delightfully easy!).

**Reply by danyeaw** [reply]
Yes, if you have all the dependencies installed, then you can `pip install gaphor`. The majority of the dev environment instructions are to set up these dependencies like GTK.
